tests/components/group/test_notify.py

```diff
@@ -90,3 +93,58 @@ def test_send_message_with_data(self):
             "title": "Test notification",
             "data": {"hello": "world", "test": "message"},
         }
+
+
+async def test_reload_notify(hass):
+    """Verify we can reload the notify service."""
+
+    assert await async_setup_component(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6103615)